num2words/lang_PT.py

```diff
@@ -21,18 +21,36 @@
 
 from .lang_EU import Num2Word_EU
 
-DOLLAR = ('dólar', 'dólares')
-CENTS = ('cêntimo', 'cêntimos')
+GENERIC_DOLLARS = ('dólar', 'dólares')
+GENERIC_CENTS = ('cêntimo', 'cêntimos')
 
 
 class Num2Word_PT(Num2Word_EU):
 
     CURRENCY_FORMS = {
-        'AUD': (DOLLAR, CENTS),
-        'CAD': (DOLLAR, CENTS),
-        'EUR': (('euro', 'euros'), CENTS),
+        'AUD': (GENERIC_DOLLARS, GENERIC_CENTS),
+        'CAD': (GENERIC_DOLLARS, GENERIC_CENTS),
+        'EUR': (('euro', 'euros'), GENERIC_CENTS),
         'GBP': (('libra', 'libras'), ('péni', 'pence')),
-        'USD': (DOLLAR, CENTS),
+        'USD': (GENERIC_DOLLARS, GENERIC_CENTS),
+        'RUB': (('rublo', 'rublos')), GENERIC_CENTS),
+        'MXN': (('peso', 'pesos'), GENERIC_CENTS),
+        'RON': ('leu', GENERIC_CENTS),
```

Comment:
This line doesn't match the other ones at all. Shouldn't the first element of the tuple be a tuple itself? 
```
        'RON': ('leu', GENERIC_CENTS),
```
Otherwise, the last line in this:
https://github.com/savoirfairelinux/num2words/blob/20a7e3a8631239605c7c227cd789c8d0214bc1e1/num2words/lang_PT.py#L248-L254
Will be `"e"`, which is obviously wrong and unintended.

Reply:
In portuguese (BR) the plural 'leu' seems to be 'lei', but in portuguese (PT) (https://repositorio.ul.pt/bitstream/10451/9948/1/ulfl139260_tm.pdf), so it should be like this, you are right (https://pt.wikipedia.org/wiki/Leu_romeno):
'RON': (('leu', 'leus'), ('ban', 'banes')),